addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4761,7 +4762,33 @@ registry.sizing = SnippetOptionWidget.extend({
                 resizeResolve();
                 self.trigger_up("enable_loading_effect");
 
+                // Check whether there has been a resizing.
                 if (directions.every(dir => dir.begin === dir.current)) {
+                    const mouseupTime = new Date();
+                    // Mouse move duration in milliseconds.
+                    const mouseHeldDuration = mouseupTime - mousedownTime;
```

Comment:
"mouse held" instead of "mouse move", since if we are here, the mouse surely didn't move (a lot) :stuck_out_tongue: 
Also I personally prefer saying "mouse pressed" instead of "mouse held", but it is just my opinion, so feel free to keep it like that :wink:

Reply:
> "mouse held" instead of "mouse move", since if we are here, the mouse surely didn't move (a lot) 😛 Also I personally prefer saying "mouse pressed" instead of "mouse held", but it is just my opinion, so feel free to keep it like that 😉

I'm going to leave it as it is because IMO "pressed" might more easily suggest a click.